### PR TITLE
Update google-ads-event-streaming.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/google-ads-event-streaming.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ Amplitude Data's Google Ads integration enables you to stream your Amplitude eve
 
 - You must enable this integration in each Amplitude project you want to use it in.
 - Amplitude sends custom events using Amplitude `event_type` as event name.
+- This integration uses the [Upload Click Conversions API](https://developers.google.com/google-ads/api/docs/conversions/upload-clicks)).
```
Comment (markzegarelli) [inline docs/data/destinations/google-ads-event-streaming.md:13]:
```suggestion
- This integration uses Google's [Upload Click Conversions API](https://developers.google.com/google-ads/api/docs/conversions/upload-clicks)).
```

File: docs/data/destinations/google-ads-event-streaming.md [hunk 2]
```diff
@@ -10,6 +10,8 @@ Amplitude Data's Google Ads integration enables you to stream your Amplitude eve
...
+- A user property **Google Click ID (GLID** or **initial_gclid** is required in order for Amplitude to create a click conversion and pass that event to Google Ads. A Google Click ID (GCLID) is a parameter passed in the URL with ad clicks, to identify the campaign and other attributes of the click associated with the ad for ad tracking and campaign attribution. In Google Ads, this is enabled by turning on the auto-tagging setting. Google Ads relies on the gclid (Google Click ID) parameter to track user interactions with ads and attribute conversions back to specific ad clicks. When a user clicks on a Google Ads ad, a unique gclid value is generated and appended to the URL. This gclid value is used to identify the click and link it to subsequent conversions.
```
Comment (markzegarelli) [inline docs/data/destinations/google-ads-event-streaming.md:14]:
```suggestion
- This integration requires a Google Click ID (`gclid` or `initial_gclid` to enable Amplitude to create the click conversion and pass that event to Google Ads. Google Ads relies on the `gclid` parameter to track user interactions with ads and attribute conversions back to specific ad clicks. When a user clicks an ad served through Google Ads, Google Ads generates a unique `gclid` value and appends it to the URL. This parameter identifies the click, and links it to subsequent conversions.
```

File: docs/data/destinations/google-ads-event-streaming.md [hunk 3]
```diff
@@ -32,9 +34,9 @@ After you create the destination, you must configure the settings.
 #### Configure settings
 
 1. On the **Settings** tab, click **Edit**.
-2. Enter your [**Google Ads Customer ID**](https://support.google.com/google-ads/answer/1704344?hl=en).
-3. Enter your [**Google Ads Conversion Action ID**](https://support.google.com/google-ads/thread/105330243?hl=en&sjid=5504033552721490234-EU).
-4. Enter your [**Base64 encoded Google Cloud Service Account**](https://developers.google.com/google-ads/api/docs/oauth/service-accounts).
+2. Enter your [**Google Ads Customer ID**](https://support.google.com/google-ads/answer/1704344?hl=en). A Google Ads Customer ID is a unique identifier assigned to each advertiser or business that uses Google Ads, which is Google's online advertising platform. This ID helps Google track and manage accounts, campaigns, and billing information for advertisers. It is essential for accessing and managing your Google Ads account.
```
Comment (markzegarelli) [inline docs/data/destinations/google-ads-event-streaming.md:37]:
```suggestion
2. Enter your [**Google Ads Customer ID**](https://support.google.com/google-ads/answer/1704344?hl=en). A Google Ads Customer ID is a unique identifier assigned to each advertiser or business that uses Google Ads. This ID helps Google track and manage accounts, campaigns, and billing information for advertisers.
```

File: docs/data/destinations/google-ads-event-streaming.md [hunk 4]
```diff
@@ -32,9 +34,9 @@ After you create the destination, you must configure the settings.
...
+3. Enter your [**Google Ads Conversion Action ID**](https://support.google.com/google-ads/thread/105330243?hl=en&sjid=5504033552721490234-EU). A Google Ads Conversion Action ID is a unique identifier associated with a specific conversion action within your Google Ads account. In Google Ads, a conversion action represents a desired action that you want visitors to your website to take, such as making a purchase, filling out a contact form, or signing up for a newsletter. The Conversion Action ID helps you track and measure the success of these actions and their associated campaigns.
```
Comment (markzegarelli) [inline docs/data/destinations/google-ads-event-streaming.md:38]:
```suggestion
3. Enter your [**Google Ads Conversion Action ID**](https://support.google.com/google-ads/thread/105330243?hl=en&sjid=5504033552721490234-EU). This is a unique identifier associated with a specific conversion action in Google Ads, and represents a desired action that you want your visitors or users to take. For example, make a purchase, submit a contact form, or sign up for a newsletter.
```

File: docs/data/destinations/google-ads-event-streaming.md [hunk 5]
```diff
@@ -32,9 +34,9 @@ After you create the destination, you must configure the settings.
...
+4. Enter your [**Base64 encoded Google Cloud Service Account**](https://developers.google.com/google-ads/api/docs/oauth/service-accounts). A Google Cloud Service Account, also known as a Service Account, is a special type of Google account that is used for server-to-server interactions and authentication within Google Cloud Platform (GCP) services. It's typically used for granting permissions and access to resources like Google Cloud Storage, Google Cloud Compute Engine, or other GCP services. Service Accounts have associated credentials (private keys or OAuth tokens) that are used for authentication when interacting with GCP services programmatically.
```
Comment (markzegarelli) [inline docs/data/destinations/google-ads-event-streaming.md:39]:
```suggestion
4. Enter your [**Base64 encoded Google Cloud Service Account**](https://developers.google.com/google-ads/api/docs/oauth/service-accounts). A Google Cloud Service Account, also known as a Service Account, is a special type of Google account used for server-to-server interactions and authentication within Google Cloud Platform (GCP) services. It's can grant permissions and access to resources like Google Cloud Storage, Google Cloud Compute Engine, or other GCP services. Service Accounts have associated credentials (private keys or OAuth tokens) for authentication when you interact with GCP services programmatically.
```

File: docs/data/destinations/google-ads-event-streaming.md [hunk 6]
```diff
@@ -48,6 +50,12 @@ The final step is enabling the destination. You must enable the destination to s
 2. Toggle **Status** from **Disabled** to **Enabled**.
 3. Save when finished.
 
+## Use Cases
+
+1. **Accurate Conversion Tracking:** The integration allows you to track conversions more accurately by sending Amplitude event data directly to Google Ads. This ensures that you can attribute conversions to specific user actions and events, helping you understand which aspects of your campaigns are driving results.
+2. **Optimizing Campaigns:** With real-time data from Amplitude, you can optimize your Google Ads campaigns based on user behavior and engagement. This means you can make data-driven decisions to allocate your budget to the most effective keywords, ad creatives, and targeting options.
+3. **Personalization:** By leveraging Amplitude's event data, you can create highly personalized ad campaigns. For example, you can use the data to target users who have completed specific actions on your website or app with tailored ad messages or offers.
```
Comment (markzegarelli) [inline docs/data/destinations/google-ads-event-streaming.md:57]:
```suggestion
3. **Personalization:** With Amplitude's event data, you can create highly personalized ad campaigns. For example, you can use the data to target users who have completed specific actions on your website or app with tailored ad messages or offers.
```

**Review by markzegarelli — CHANGES_REQUESTED** [review]

**Comment by Brandoncharleskhoo1** [comment]
Hi @markzegarelli , thanks for reviewing the document and suggesting those changes. I've implemented those suggested changes but I'm still unable to merge. Can you please help me to merge?

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**